conjur/login/credential_data.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+"""
+CredentialData module
+
+This module represents an object that holds netrc data
+"""
+
+# pylint: disable=too-few-public-methods
+class CredentialData:
```

Comment:
Credential**s** is the correct term, I think.
Make sure to reflect this change to all the code.
```suggestion
class CredentialsData:
```